server/pulp/server/managers/consumer/content.py

```diff
@@ -41,7 +24,7 @@ def install(self, id, units, options={}):
         @type options: dict
```

Comment:
On 01/08/2015 04:55 PM, Austin wrote:
> Do you want to switch over to |:param:| during style changes?

No. Once I finish PEP-8'ing our code, PEP-257 is next!